devfile.yaml

```diff
@@ -0,0 +1,51 @@
+metadata:
+  name: kubernetes-image-puller
+projects:
+  - name: kubernetes-image-puller
+    source:
+      location: 'https://github.com/che-incubator/kubernetes-image-puller.git'
+      type: git
+      branch: master
+    clonePath: src/github.com/redhat-developer/kubernetes-image-puller
```

Comment:
not sure if this line is expected

Reply:
this clones it into $GOPATH which I think will be expected by the average go user